app/services/DataParser.scala

```diff
@@ -16,26 +16,12 @@
 
 package services
 
-import java.util.concurrent.TimeUnit
-
-import config.ApplicationConfig
-import javax.inject.{Inject, Singleton}
 import javax.xml.parsers.SAXParserFactory
-import metrics.Metrics
 import models._
 import play.api.Logger
-import play.api.mvc.Request
-import services.ERSTemplatesInfo._
-import services.audit.AuditEvents
-import services.validation.ErsValidator
-import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.services.validation.{DataValidator, ValidationError}
 import utils.ErrorResponseMessages
 
-import scala.collection.mutable.ListBuffer
-import scala.concurrent.duration.Duration
-import scala.concurrent.{Await, ExecutionContext, Future}
-import scala.util.{Failure, Success, Try}
+import scala.util.Try
 import scala.xml._
 
 trait DataParser {
```

Comment:
Line 43: Commented code. Remove if not required.

Reply:
that's gone